libsol/transaction_printers.c

```diff
@@ -634,5 +638,20 @@ int print_transaction(const PrintConfig* print_config,
         infos_length--;
     }
 
+    if (infos_length > 1) {
+        // Iterate over infos and print compute budget instructions and offset pointers
+        // Handle ComputeBudget instructions first due to teh limitations of the print_transaction_nonce_processed.
```

Comment:
not sure what this comment means

Reply:
Main problem is inside `transaction_printers.c` file. We have there hardcoded cases of instructions in one big switch statement. Because of it, handling all cases related to the ComputeBudget would introduce a lot of low quality and unreadable code. It can be solved by iterating over CB instructions first, this way we can not care about the amount nor the order.

This is one of the problems that blocked us with the token2022 extensions. To many combinations to handle

Reply:
I can extend this comment if you want or just delete it, it's up to you